gammapy/astro/darkmatter/spectra.py

```diff
@@ -112,3 +114,72 @@ def table_model(self):
         dN_dE = dN_dlogx / (energies * np.log(10))
 
         return TableModel(energy=energies, values=dN_dE, values_scale="lin")
+
+
+class DMAnnihilModel(SpectralModel):
```

Comment:
Maybe rename to `DMAnnihilation`? We typically don't have `...Model` as a suffix for in Gammapy...